uvls/src/ide/completion.rs

```diff
@@ -990,6 +991,22 @@ fn compute_completions_impl(
                     &ctx,
                 ))
             }
+             // complete all files and dic which are not loaded 
+            for (path, name, node) in snapshot.fs().all_sub_files(origin, & ctx.prefix,ctx.postfix.clone()) {
+                let len = path.as_str().chars().filter(|c| c == &'.').count();
+                top.push(CompletionOpt::new(
+                    match node {
+                        FSNode::Dir => CompletionKind::Folder,
+                        _ => CompletionKind::File,
+                    },
+                    name,
+                    path.clone(),
+                    len,
+                    TextOP::Put(path),
```

Comment:
I think you should replace the slashes with dots here for proper completion

Reply:
yes should actually be so, just made a mistake that should be fixed in the next push